src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -5822,6 +5842,10 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
                 using_func_attr = true;
                 return;
             }
+            if (attr == "args") {
+                using_args_attr = true;
```

Comment:
Where is this variable getting reset to `false`? If it isn't, then I think that's a bug.

Reply:
Addressed.